src/analytics/models/dashboard.py

```diff
@@ -0,0 +1,46 @@
+import app 
+from helpers import mongo_helpers
```

Comment:
Ideally, this is where we are supposed to define all our collection schemas. As seen in ATLAS, we have: 

1. device_daily_historical_averages
2. device_daily_measurements
3. device_hourly_measurements
4. device_raw_measurements
5. devices
6. division
7. monitoring_site

So those are the tables (collections) whose schemas are supposed to be under this `models` directory. 
More like:
```
from .db import db

class Movie(db.Document):
Division = db.StringField(required=True, unique=True)
SiteName = db.StringField(required=True, unique=True)
DeviceCode = db.StringField(required=True, unique=True)
Latitude = db.StringField(required=True, unique=True)
Longitude = db.StringField(required=True, unique=True)
Organisation = db.StringField(required=True, unique=True)
LocationCode = db.StringField(required=True, unique=True)
Parish = db.StringField(required=True, unique=True)
LatestHourlyMeasurement = db.ListField(db.StringField(), required=True)
```

Reply:
And then within the same class, we could add some "static" or "instance" functions which are currently under `helpers/mongo_helpers`